### ISSUE TITLE
Heading levels should only increase by one

### ISSUE THREAD

**Issue by markteekman** [incluud/accessible-astro-starter#153]
For /blog and /portfolio, the cards should have an `h2` instead of the default `h3`.

**Comment by markteekman** [comment]
Fixed in #160
